### DISCUSSION TITLE
How to change font css?

### DISCUSSION THREAD

**Discussion by WarMachine-282** [Happy-Coding-Clans/vue-easytable#511] [Q&A]
Hello, I'm currently using your table for a vue2 application. I'm running into problems with changing font size, family, and weight. It does not show up in the documentation. I was wondering if it is possible and if so then how? I've tried everything I can think of.

**Comment by huangshuwei** [comment]
First of all, This componet supports theme custom. Font size can be set through this feature. refer to https://happy-coding-clans.github.io/vue-easytable/#/en/doc/theme?anchor=theme-custom

In addition, there is no font  family set for this component. The font family of the component depends on your project.
